k8s/k8s.go

```diff
@@ -208,6 +208,12 @@ func CmdAddK8s(ctx context.Context, args *skel.CmdArgs, conf types.NetConf, epID
 			logger.Error(e)
 			return nil, e
 		case ipAddrsNoIpam != "":
+			if !conf.FeatureControl.IPAddrsNoIpam {
+				e := fmt.Errorf("can't honor 'ipAddrsNoIpam' request: ipAddrsNoIpam is not enabled")
```

Comment:
I'd recommend this for an error message:

`"requested feature is not enabled: cni.projectcalico.org/ipAddrsNoIpam"`

Reply:
What about `requested feature is not enabled: ip_addrs_no_ipam`, since that's what the name of it is in the CNI config?

Reply:
Yeah, I did consider that as well.

I went with the annotation since this is in the Kubernetes code and that's the way that Kubernetes users will interact with that feature. I'm OK with using `ip_addrs_no_ipam` but I do prefer the full annotation.